Proposed text for 'Guidance for closing issues' (#137)
camaraproject/Governance: CONTRIBUTING.md

Please provide as much context as possible when you open an issue. The information you provide must be comprehensive enough to reproduce that issue for the assignee. Therefore, contributors may use but aren't restricted to the issue template provided by the project maintainers.

### Guidance for closing issues
* As per the [ProjectCharter](https://github.com/camaraproject/Governance/blob/5fc802713d71a51da64136f692b16ed620eeffa5/ProjectCharter.md), The default decision making mechanism for the CAMARA Project is [lazy consensus](https://couchdb.apache.org/bylaws.html#lazy). This means that any decision on technical issues is considered supported by the team as long as nobody objects based on substantiated technical grounds.
* Once consensus is achieved, an issue may be closed by the original author or by a maintainer in the sub-project.

> * Closing an issue should include a comment on how it was resolved, or reference to consensus that the issue can not be resolved.

Reviewer: We have the principle of 'lazy consensus' which means that there is not always a document consensus which can be referenced.

```suggestion
* Closing an issue should include a comment on how it was resolved or why it won't be resolved
```

> * Issues that are considered stale/inactive may be closed, but must include a reference to minutes where it has been agreed to be a stale issue. 

Reviewer: Again, I don't think that especially stale/inactive issues need or even can always include a reference to an explicit decision. Sometimes Maintainers just have to clean up. Propose to delete the line as covered by the previous one.

Author: Yes, the previous line can cover this ('closing because no update since x weeks'). Will remove this one.

> * Maintainers may not close their own issues without reference to consensus achieved.

Reviewer: Why not? If even a non-Maintainer is allowed to do that? (see line 33). Propose to omit the line.

Author: Yes, will omit the line, as line 33 covers it.

* A sub-project participant may object to closure of an issue if they believe consensus was not reached, in which case they may reopen the issue, stating they believe consensus has not been reached.

> * Closure of an issue may require closure of a Pull Request which only fixed that single (now closed) issue.

Author: Agreed and committed

> * Closure of an issue may require closure of a Pull Request which only fixed that single (now closed) issue.

Author: Working on them between meetings :)  Will have it all done end of day
